src/services/user.service.ts

```diff
@@ -1,9 +1,58 @@
-import userRepository, { getAdminUser } from "../repositories/user.repository";
+import { UserLeleka } from "../models/UserLeleka";
+import {
+  create,
+  deleteOne,
+  getList,
+  getUserById,
+  updateOne,
+} from "../repositories/user.repository";
 
-export const getUserById = (id: string) => {
-  userRepository.findById(id);
+export const listUsers = async () => {
+  try {
+    const result = await getList();
+    if (result) {
+      return result;
+    }
+  } catch (error) {
+    throw new Error("Error");
+  }
 };
 
-export const getAdmin = () => {
-  getAdminUser();
+export const getUser = async (id: string) => {
+  try {
+    const result = await getUserById(id);
+    if (result) {
+      return result;
+    }
+  } catch (error) {
+    throw new Error("Error");
```

Comment:
should be more detalised errors)
```suggestion
    throw new Error(error.message);
```